contracts/registry/IdentityRegistry.sol

```diff
@@ -12,12 +12,12 @@ import "@onchain-id/solidity/contracts/Identity.sol";
 
 contract IdentityRegistry is IIdentityRegistry, AgentRole {
     // mapping between a user address and the corresponding identity contract
-    mapping(address => Identity) public identity;
+    mapping(address => Identity) private identity;
 
```

Comment:
Could it be possible to add a mapping between Identity and Address

Reply:
No, because an identity can be linked to multiple wallets

Reply:
At least, it was the original idea

Reply:
we put the addresses in a claim on the onchainID, we can retrieve it from there easily